app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,60 @@
+class CoursesController < ApplicationController
+  before_action :set_course, except: %i(index new create)
+
+  def index
+    @course = Course.order(:name).page params[:page]
```

Comment:
You should write order to scope in model and call it here

Reply:
i have wrote in model and it worked without adding coding at here